### PR TITLE
	modified documentation of pupil core, few typos and fix broken link 

### PR DESCRIPTION
**Pull request by mikelgg93**
misc. add articles, change eye ball to eyeball.

	modified:   src/core/academic-citation.md
    Link to how to cite invisible was broken (404), modified, however, it is hard-coded to the URL not using local ones, feel free to change it.

	modified:   src/core/software/pupil-capture.md
    On recording it refers to a "blue circular R", r is not blue, so removed.

	modified:   src/core/software/pupil-player.md
    e - hotkey is not shown as "e" but as download icon. Changed to reflect this.

    mgg 220705

### CODE REVIEW

**Review by papr — CHANGES_REQUESTED** [review]
Hey @mikelgg93! I am happy to see that you were able to contribute so early after having just started! Content changes look great, thanks for catching those!

I have added a few formal notes below that are otherwise not documented (yet).

- PR description - I like the high-level explanation for each file 👌🏻 I wonder, did you generate the PR description manually or automatically, e.g. by squashing multiple commits? For the next point, I will assume that multiple commits got squashed.
- PR/branch history - A very opinionated topic. We don't have any rigid guidelines on this but typically the PR branch would preserve the commit history instead of squashing them for a "cleaner" history. This enables a commit-by-commit review which can be helpful with more complex PRs.
- PR title - It looks like the title was auto-generated from your description. Please take care of choosing a descriptive title. I know this is not always easy with PRs that just change a few small things.
- File specific notes, see below

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,8 @@ contents/css
 contents/js/*.js
 assets/rev_manifest
 rev_manifest.json
+changelog.txt #mgg just to write the commit message
+
```
[the remark this answers is not in the document]
Comment (mikelgg93) [reply .gitignore]: Thanks! didn't know about it.

**Comment by mikelgg93** [comment]
Added the changes req.
Regarding the PR description, I did not squash them, but manually generated the description using `git commit -eF changelog.txt`. But I will definetly take that into account for the future, also for the title.
Thank you!

**Comment by papr** [comment]
> manually generated the description using `git commit -eF changelog.txt`.

Nice, I didn't know about this option!

Thanks for following up on the review so quickly, and thanks again for the contribution. 🙏🏻

**Review by papr — APPROVED** [review]

**Pull request merged**